app/views/access/login.html.erb

```diff
@@ -0,0 +1,23 @@
+ <% if !flash[:notice].blank? %>
```

Comment:
a good way of managing flashes is to have one partial view that shows any flash message and all views rely on that single partial view